backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -379,6 +386,56 @@ export const deleteCampSessionsDtoValidator = async (
   return next();
 };
 
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable-next-line import/prefer-default-export */
+export const updateCampSessionsDtoValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const campSessions = req.body.updatedCampSessions;
+
+  if (!campSessions) {
+    return res.status(400).send("updatedCampSessions is a required field");
+  }
+
+  for (let index = 0; index < campSessions.length; index += 1) {
```

Comment:
any reason not to use the inbuilt `forEach()` function?

Reply:
iirc there was the `forEach` linter problems, maybe https://github.com/github/eslint-plugin-github/issues/86?

Reply:
ok, no big deal